tests/failpoints/cases/test_stale_read.rs

```diff
@@ -526,3 +529,47 @@ fn test_stale_read_during_merging_2() {
 
     fail::remove(leader_commit_prepare_merge_fp);
 }
+
+#[test]
+fn test_stale_read_with_ts0() {
```

Comment:
If no failpoint is used, then it should be put in integration directory.

Reply:
Done